db/migrate/20241025084443_create_payments.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePayments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :payments do |t|
+      t.decimal :amount, precision: 10, scale: 2, null: false 
```

Comment:
For amount we should use `money-rails` gem. It is already there in the app so you can see `Service` model to understand how it is tied in the app.

Reply:
@rt4914 `money-rails` has been added with proper setup in Migration table and Model.